src/Eio/Poll.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace React\Filesystem\Eio;
+
+use React\EventLoop\ExtUvLoop;
+use React\Filesystem\Node\FileInterface;
+use React\Filesystem\PollInterface;
+use React\Filesystem\Stat;
+use React\Promise\Promise;
+use React\Promise\PromiseInterface;
+use RuntimeException;
+use React\EventLoop\LoopInterface;
+use React\Filesystem\Node;
+use UV;
+use function React\Promise\all;
```

Comment:
It looks like there are some leftovers here and in other files, exemplary here:

```suggestion
use React\Filesystem\PollInterface;
use React\EventLoop\LoopInterface;
```